iroha_client/tests/tests/tests/permissions.rs

```diff
@@ -123,7 +123,7 @@ fn permissions_disallow_asset_burn() {
         rejection_reason,
         &PipelineRejectionReason::Transaction(TransactionRejectionReason::NotPermitted(
             NotPermittedFail {
-                reason: "Can\'t burn assets from another account.".to_owned(),
+                reason: "Failed to pass first check with Can\'t burn assets from another account. and second check with Account does not have the needed permission token: PermissionToken { name: \"can_burn_user_assets\", params: {\"asset_id\": Id(AssetId(Id { definition_id: DefinitionId { name: \"xor\", domain_name: \"wonderland\" }, account_id: Id { name: \"bob\", domain_name: \"wonderland\" } }))} }..".to_owned(),
```

Comment:
Wow, such long string. May be can be moved to constant? You can use `concat!` there, because may be someone doesn't use line wrapping in text editor)